src/Scenario/ClientGenerator.php

```diff
@@ -9,9 +9,18 @@
 use TelegramOSINT\Client\InfoObtainingClient\InfoClient;
 use TelegramOSINT\Client\StatusWatcherClient\StatusWatcherCallbacks;
 use TelegramOSINT\Client\StatusWatcherClient\StatusWatcherClient;
+use TelegramOSINT\Exception\TGException;
 
 class ClientGenerator implements ClientGeneratorInterface
 {
+    /** @var string */
+    private $envName;
+
+    public function __construct(string $envName = 'AUTHKEY')
```

Comment:
Would be better make it mandatory, not optional

Reply:
Ok, made mandatory, env var is now selected at 3rd/4th level.